docs/static/persistent-queues.asciidoc

```diff
@@ -128,22 +131,35 @@ events.
 [[durability-persistent-queues]]
 ==== Controlling Durability
 
+// I'm not sure about this particular description of durability. Might need some help.
+Durability is a property of storage writes that indicate the availability of
+that data after it was written.
+
 When the persistent queue feature is enabled, Logstash will store events on
-disk. The persistent queue exposes the trade-off between performance and
-durability by providing the following configuration options:
+disk. Logstash commits to disk in a mechanism we call checkpointing.
+
+To discuss durability, we need to introduce a few details about how the persistent queue is implemented.
```

Comment:
Hmmm....we're sort of started down this road earlier. I wonder if all the info about pages could be described in one section? Only the most dedicated readers are going to want to plow through these details.